CI failure is real. Checkstyle issue

```
[INFO] --- maven-checkstyle-plugin:3.1.1:check (validate) @ jenkins-core ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Jenkins main module 2.276-SNAPSHOT:
[INFO] 
[INFO] Jenkins main module ................................ SUCCESS [  9.539 s]
[INFO] Jenkins BOM ........................................ SUCCESS [  1.680 s]
[INFO] Jenkins cli ........................................ SUCCESS [ 35.797 s]
[INFO] Jenkins core ....................................... FAILURE [ 40.770 s]
[INFO] Jenkins war ........................................ SKIPPED
[INFO] Tests for Jenkins core ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:31 min
[INFO] Finished at: 2021-01-21T23:19:00+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check (validate) on project jenkins-core: Failed during checkstyle configuration: Exception was thrown while processing /home/tiste/dev/github/jenkinsci/jenkins/core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java: IllegalStateException occurred while parsing file /home/tiste/dev/github/jenkinsci/jenkins/core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java. /home/tiste/dev/github/jenkinsci/jenkins/core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java:127:6: expecting RCURLY, found '*' -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :jenkins-core
```

--- a/core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java
+++ b/core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java
@@ -123,7 +123,7 @@ public class FileFingerprintStorage extends FingerprintStorage {
 
     /**
      * Saves the given Fingerprint in local XML-based database.
-     */
+     *
      * @param fp Fingerprint file to be saved.
      */
     @Override

